torch_xla/csrc/ops/index_ops.cpp

```diff
@@ -58,6 +60,21 @@ std::vector<at::Tensor> ExpandByteTensors(const at::Tensor& self,
   return result;
 }
 
+struct IndexAdjacencyInfo {
+  bool contiguous_non_null;
+  xla::int64 start_dim;
+};
+
+IndexAdjacencyInfo GetIndexAdjacencyInfo(at::TensorList indices) {
```

Comment:
Small comment would help.

Reply:
Also, mentioning the aten source this is coming from, can speed up the review.

Reply:
Done.